src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php

```diff
@@ -64,8 +64,10 @@ protected function generateUrl($route, $parameters = array(), $referenceType = U
      */
     protected function forward($controller, array $path = array(), array $query = array())
     {
+        $request = $this->container->get('request_stack')->getCurrentRequest();
+        $path['_forwarded'] = $request->attributes;
```

Comment:
@xabbuh, the point is not to catch subrequests as profiler already does, we can see them in the request panel.
This feature is about catching the "main" forward", using this special method, as it should be used only once in most use case to quickly click the profile token link of the original request in the wdt.

But I can try to go further, how would you imagine the presentation of sub requests token in the wdt ?